ui/components/app/modals/keyring-snap-removal-modal/keyring-snap-removal-result-modal.tsx

```diff
@@ -15,11 +15,11 @@ import {
   IconName,
   IconSize,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
 import { getKeyringSnapRemovalResult } from '../../../../selectors';
 
```

Comment:
No story to easily compare UI but can be confident there are no changes from other UI